run.sh

```diff
@@ -52,8 +54,14 @@ rm -rf /elasticsearch/modules/x-pack/x-pack-ml
 
 # run
 if [[ $(whoami) == "root" ]]; then
-    chown -R elasticsearch:elasticsearch $BASE
-    chown -R elasticsearch:elasticsearch /data
+    #chown -R elasticsearch:elasticsearch $BASE
+    #chown -R elasticsearch:elasticsearch /data
+	echo "Changing ownership of $BASE folder"
+	find . -type f -print0 | xargs -0 chown elasticsearch:elasticsearch $BASE
```

Comment:
@pires This looks wrong to me. The way I read this: for every file found somewhere (`.`) run `chown elasticsearch:elasticsearch /elasticsearch`. Most likely it should have been:
```
find "$BASE" -type f -print0 | xargs -0 chown elasticsearch:elasticsearch
```
(the same for the find/chown line below)

Reply:
You are correct! And you have found a bug.

Since the script executes in /elasticsearch folder, find . will only list its files. The $BASE folder items will be affected, but the /data folder items will never be changed. Since /elasticsearch only has about 270 files, it worked fast, so that was the speed up I noticed :(

After looking a bit more into find syntax, I got what I wanted from the start:

```
find /data -type f -not -user elasticsearch -print0 | xargs -0 chown elasticsearch:elasticsearch /data
```

This will actually look only for files that do not have the user elasticsearch owner and feed them to a batch execution of chown. This works out to be fast (30s for 150K files) and effective as well :)

And I have to do another pull request...